### PR TITLE
remove duplicate chains in bounty/new

### PR DESCRIPTION
**Pull request by chibie**
##### Description

Remove duplicate chains - BTC, PayPal, Harmony

image

### CODE REVIEW

File: app/dashboard/templates/bounty/new_bounty.html [hunk 1]
```diff
@@ -136,17 +124,13 @@ <h1 class="text-center">Fund Issue</h1>
...
                   <label class="btn btn-radio chain-btn d-flex align-items-center mr-2 mb-2 font-weight-bold py-2 px-4" :class="{'active': chainId === '42220'}">
                     <input type="radio" name="bounty_chain" id="42220_chain" value="42220" v-model="chainId"><img class="mr-2" src="{% static 'v2/images/chains/celo.svg' %}" alt="" width="16"> Celo
                   </label>
-                  {% endif %}
 
-                  {% if is_staff %}
                   <label class="btn btn-radio chain-btn d-flex align-items-center mr-2 mb-2 font-weight-bold py-2 px-4" :class="{'active': chainId === '717171'}">
                     <input type="radio" name="bounty_chain" id="717171_chain" value="717171" v-model="chainId"> Other
                   </label>
-                  {% endif %}
```
Comment (thelostone-mc) [inline app/dashboard/templates/bounty/new_bounty.html:149]: This needs to be in staff

Comment (chibie) [reply app/dashboard/templates/bounty/new_bounty.html:149]: other?

Comment (octavioamu) [reply app/dashboard/templates/bounty/new_bounty.html:149]: yes

**Comment by octavioamu** [comment]
harmony is repeated @chibie

File: app/dashboard/templates/bounty/new_bounty.html (hunk 1 above)
Comment (chibie) [reply app/dashboard/templates/bounty/new_bounty.html:149]: done

**Review by octavioamu — APPROVED** [review]

**Pull request merged**